public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
+  └─ license.md
+:marked
+  Functionally, it's an `index.html` and two TypeScript files in an `app/` folder.
+  We can handle that!
+  
+  Of course we won't build  many apps that only run in plunker. 
+  Let's follow a process that's closer to what we'd do in real life.
+  
+  1. Set up our development environment
+  1. Write the Angular root component for our main app
+  1. Bootstrap it to take control of the main web page
+  1. Write the main page (`index.html`)
+  
+.l-sub-section
+  :marked
+    We really can build the QuickStart from scratch in five minutes 
+    if we follow the instructions and ignore the commentary.
+    
+    Most of us will be interested in the "why" as well as the "how" and that will take longer.
+:marked
 
 .l-main-section
-  :marked
-    ## Run it!
-
-    We need a static file server to serve our application to the browser.
-
-  .l-sub-section
-    :marked
-      Don't have a static file server handy?  Let's install one of our favorites
-      called [live-server](https://www.npmjs.com/package/live-server "Live-server")
-      with the **npm package manager**.
-
-      Don't have npm?
-      [Get it now](https://docs.npmjs.com/getting-started/installing-node "Installing Node.js and updating npm")
-      because we're going to use it now and repeatedly throughout this documentation.
-
-      Once you have `npm` installed, open a terminal window and enter
-
-    pre.prettyprint.lang-bash
-      code npm install -g live-server
-
-  :marked
-    Open a terminal window and enter
+:marked
+  ## Development Environment
+  
+  We'll need a place to stand (the application project folder), some libraries, 
+  some TypeScript configuration and the TypeScript-aware editor of your choice.
+
+  ### Create a new project folder
+  
+code-example(format="").
+  mkdir angular2-quickstart
+  cd    angular2-quickstart
+:marked
+  ### Add the libraries we need
+  
+  We recommend the **npm** package manager for acquiring and managing our development libraries.
 
-  pre.prettyprint.lang-bash
-      code live-server
+.l-sub-section
   :marked
-    In a few moments, a browser tab should open and display
-
-  figure.image-display
-    img(src='/resources/images/devguide/quickstart/my-first-app.png' alt="Output of quickstart app")
+    Don't have npm?
+    [Get it now](https://docs.npmjs.com/getting-started/installing-node "Installing Node.js and updating npm")
+    because we're going to use it now and repeatedly throughout this documentation.
+:marked
+  Add a **package.json** file to the project folder and copy/paste the following:
++makeJson('quickstart/ts/package.1.json', null, 'package.json')(format=".")  
 
+.l-sub-section
   :marked
-    Congratulations!  We are in business.
-
-  .alert.is-helpful
-    :marked
-      If you see `Loading...` displayed instead, see the
-      [Browser ES6 support appendix](#es6support).
-
-.l-main-section
+    Itching to know the details? We explain in the [appendix below](#package-json)  
```

Comment:
Clicking this link will hide the linked section header. This is a problem present throughout `angular.io` though, because there is no padding/margin to offset the navbar at the top of the main section.

So we get this:
![image](https://cloud.githubusercontent.com/assets/4172079/11746679/0e9d2220-a017-11e5-8e1b-d0f635a120b7.png)

While we should get this:
![image](https://cloud.githubusercontent.com/assets/4172079/11746779/75b1eed2-a017-11e5-852f-8915f1e37969.png)

I'll open an issue for it. Here #477.